webapp/templates/team/base.html.twig

```diff
@@ -14,6 +14,17 @@
             $('body').on('submit', 'form[name=team_clarification]', function () {
                 return confirm("Send clarification request to Jury?");
             });
+
+            /* Show the notification options if the browser supports it */
+            if ('Notification' in window) {
+                $('#notify_enable').click(enableNotifications);
+                if (getCookie('domjudge_notify') != 1) {
+                    $('#notify_enable').removeClass('d-none');
+                }
+            }
+            updateClarifications();
+            setInterval(updateClarifications, 20000);
```

Comment:
Will this re-add all unread notifications every 20 seconds if teams dismiss them? That's maybe not what we want?

Reply:
The sendNotification function use local storage to track which notification has been sent, so the same clar won't be popup twice.

Reply:
Ah my bad, it has been a while since I looked at the method. Then it's fine.